app/models/event_log.rb

```diff
@@ -11,7 +11,7 @@ class EventLog < ActiveRecord::Base
   UNSUCCESSFUL_LOGIN = "Unsuccessful login"
   SUSPENDED_ACCOUNT_AUTHENTICATED_LOGIN = "Unsuccessful login attempt to a suspended account, with the correct username and password"
   UNSUCCESSFUL_PASSPHRASE_CHANGE = "Unsuccessful passphrase change"
-  EMAIL_CHANGE_INITIATIED = "Email change initiated"
+  EMAIL_CHANGE_INITIATIED = "Email changed"
```

Comment:
I'm not convinced by this change of wording.  As I understand it, the email change does not necessarily take effect until it's confirmed, so this could be misleading.

I can see that this gets tricky because of the 2 different flows though (admin and used flows).  Maybe we need separate events for each flow - 2 for the user flow, and one for the admin flow.